lib/src/facts/windows/operating_system_resolver.cc

```diff
@@ -100,8 +100,8 @@ namespace facter { namespace facts { namespace windows {
         // Override default release with Windows release names
         auto version = result.release.substr(0, lastDot);
         bool consumerrel = (wmi::get(vals, wmi::producttype) == "1");
-        if (version == "6.4") {
```

Comment:
Not that I could find. We looked into this a fair amount when I originally put up the PR, and it seems they switched everything to 10.